docs/how-monty-works/observations-transforms-sensor-modules.md

```diff
@@ -36,11 +36,11 @@ For an overview of **which type of data processing belongs where please refer to
 - The output of both SMs and LMs is a State class instance containing information about pose relative to the body and detected features at that pose. This is the input that any LM expects.
 
 ## Features Extracted by the Sensor Module
-In this implementation, some features are extracted using all of the information in the sensor patch (e.g. locations of all points in the patch for point normal and curvature calculation) but then refer to the center of the patch (e.g. only the curvature and point normal of the center are returned). At the moment all the feature extraction is predefined but in the future, one could also imagine some features being learned.
+In this implementation, some features are extracted using all of the information in the sensor patch (e.g. locations of all points in the patch for surface normal and curvature calculation) but then refer to the center of the patch (e.g. only the curvature and surface normal of the center are returned). At the moment all the feature extraction is predefined but in the future, one could also imagine some features being learned.
 
-## Point Normals and Principle Curvatures
-We can use the point normal and principal curvature to define the orientation of the sensor patch. The following video describes what those represent.
-[Point Normals and Principle Curvatures](https://res.cloudinary.com/dtnazefys/video/upload/v1731342526/point_normal.mp4)
+## Surface Normals and Principle Curvatures
+We can use the surface normal and principal curvature to define the orientation of the sensor patch. The following video describes what those represent.
```

Comment:
Just because the video currently calls it point normal and that will be a point of confusion until we create a new video.

```suggestion
We can use the surface normal (previously referred to as point-normal) and principal curvature to define the orientation of the sensor patch. The following video describes what those represent.
```